docker/2.1/Dockerfile.gpu

```diff
@@ -0,0 +1,132 @@
+ && apt-get install -y --no-install-recommends --allow-unauthenticated --allow-downgrades\
+    ca-certificates \
+    cuda-command-line-tools-10-1 \
+    cuda-cufft-10-1 \
+    cuda-curand-10-1 \
+    cuda-cusolver-10-1 \
+    cuda-cusparse-10-1 \
+    #cuda-cublas-dev not available with 10-1, install libcublas instead
+    libcublas10=10.1.0.105-1 \
+    libcublas-dev=10.1.0.105-1 \
+    libcudnn7=${CUDNN_VERSION} \
+    libnccl2=${NCCL_VERSION} \
+    libgomp1 \
+    curl \
+    git \
+    wget \
+    vim \
+    build-essential \
+    zlib1g-dev \
+    python3 \
+    python3-pip \
+    python3-setuptools \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+# The 'apt-get install' of nvinfer-runtime-trt-repo-ubuntu1804-4.0.1-ga-cuda10.0
+# adds a new list which contains libnvinfer library, so it needs another
+# 'apt-get update' to retrieve that list before it can actually install the
+# library.
+# We don't install libnvinfer-dev since we don't need to build against TensorRT,
+# and libnvinfer4 doesn't contain libnvinfer.a static library.
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends nvinfer-runtime-trt-repo-ubuntu1804-5.0.2-ga-cuda10.0 \
+ && apt-get update \
+ && apt-get install -y --no-install-recommends libnvinfer6=${TF_TENSORRT_VERSION}-1+cuda10.1 \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/* 
+ #&& rm /usr/lib/x86_64-linux-gnu/libnvinfer_plugin* \
+ #&& rm /usr/lib/x86_64-linux-gnu/libnvcaffe_parser* \
+ #&& rm /usr/lib/x86_64-linux-gnu/libnvparsers*
+
+RUN ${PIP} --no-cache-dir install --upgrade \
+    pip \
+    setuptools
+
+# Some TF tools expect a "python" binary
+RUN ln -s $(which ${PYTHON}) /usr/local/bin/python
+
+# nginx + njs
+RUN apt-get update \
+ && apt-get -y install --no-install-recommends \
+    curl \
+    gnupg2 \
+ && curl -s http://nginx.org/keys/nginx_signing.key | apt-key add - \
+ && echo 'deb http://nginx.org/packages/ubuntu/ bionic nginx' >> /etc/apt/sources.list \
+ && apt-get update \
+ && apt-get -y install --no-install-recommends \
+    nginx \
+    nginx-module-njs \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+# cython, falcon, gunicorn, grpc
+RUN ${PIP} install -U --no-cache-dir \
+    awscli==1.17.11 \
+    cython==0.29.14 \
+    falcon==2.0.0 \
+    gunicorn==20.0.4 \
+    gevent==1.4.0 \
+    requests==2.22.0 \
+    grpcio==1.27.1  \
+    protobuf==3.11.1 \
+# using --no-dependencies to avoid installing tensorflow binary
+ && ${PIP} install --no-dependencies --no-cache-dir \
+    tensorflow-serving-api-gpu==2.1.0
+
+COPY ./sagemaker /sagemaker
+
+RUN curl $TFS_URL -o /usr/bin/tensorflow_model_server \
+ && chmod 555 /usr/bin/tensorflow_model_server
+
+# put tensorflow library (with only error_codes) to python dist-packages
+WORKDIR /
+RUN cd /usr/local/lib/python3.*/dist-packages/ && \
+    mv /sagemaker/tensorflow-2.1 ./tensorflow && \
+    # Delete the remaining tensorflow folder to avoid confusing python import
+    rm -rf /sagemaker/tensorflow
```

Comment:
Please re-format these lines as per the recommended format for multi-line RUN statements (as used in the other RUN statements in this dockerfile).